common/changes/@microsoft/rush/qun-sha1_2024-05-30-16-44.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Display to the user subspace name where the error occurred",
```

Comment:
```suggestion
      "comment": "Display the name of the subspace when an error is emitted because a dependency hash uses the SHA1 algorithm and the \"disallowInsecureSha1\" option is enabled.",
```